config/build_olcf_titan.sh

```diff
@@ -33,21 +33,23 @@ module load cmake/2.8.11.2
 # Set environment variables
 export FFTW_HOME=$FFTW_DIR/..
 
+CC=cc
```

Comment:
These CC and CXX variables are not exported and are only defined in this shell script. They will have no effect on the cmake process.

They should be exported or passed to cmake with  -D CMAKE_C_COMPILER=$cc (and similarly for CXX)